README.md

```diff
@@ -1,5 +1,5 @@
-[![](https://vsmarketplacebadges.dev/version-short/MS-SarifVSCode.sarif-viewer.svg)](https://marketplace.visualstudio.com/items?itemName=MS-SarifVSCode.sarif-viewer)
-[![](https://vsmarketplacebadges.dev/downloads-short/MS-SarifVSCode.sarif-viewer.svg)](https://marketplace.visualstudio.com/items?itemName=MS-SarifVSCode.sarif-viewer)
+[![](https://vsmarketplacebadges.dev/version-short/MS-SarifVSCode.sarif-viewer.png)](https://marketplace.visualstudio.com/items?itemName=MS-SarifVSCode.sarif-viewer)
```

Comment:
>png [](http://example.com/codeflow?start=79&length=3)

relatively blurrier than svg

Reply:
Doesn't seem like there is a good solution, so I'll just remove the badges entirely.